src/plugins/controls/public/options_list/components/options_list.scss

```diff
@@ -31,10 +31,12 @@
       color: $euiColorDanger;
     }
 
-    .euiFilterButton__text-hasNotification {
-      flex-grow: 1;
-      justify-content: space-between;
-      width: 0;
+    .euiFilterButton__text {
```

Comment:
Is it possible to add a custom class as opposed to overriding `.euiFilterButton__text`?

Reply:
Good call :+1: That should hopefully prevent these types of bugs in the future. Done in f1ee5ad9f666d712f076ae70f20e80ed83862f1a